ngraph_creator/operations/src/Conv_2d.cpp

```diff
@@ -244,12 +267,18 @@ std::shared_ptr<ngraph::Node> Conv_2d::createNode() {
         convNode, biasNode, ngraph::op::AutoBroadcastType::NUMPY);
     outputNode = applyActivation(outputNode, activationFn);
 
+    if (!useNchw) {
+        outputNode = transpose(NCHW_NHWC, outputNode);
+        mNgraphNodes->setForcedNchw(mDefaultOutputIndex, false);
```

Comment:
This is negating the use of *ForcedNchw flag. We may rather get this usage removed.